Features Added

ReyAntonio12 opened the pull request:
Adding linters to be used in this project

ReyAntonio12 committed:
Add features to project, linters added to be working with pull request

ReyAntonio12 committed:
Add web index.html to show the base for my project

ReyAntonio12 committed:
Add styles sheet to project, no linters errors have been shown in this commit

ReyAntonio12 committed:
Edit css file to fix identation and whitespace issues from stylelint

ReyAntonio12 committed:
Add MIT document to project and edit readme file

ReyAntonio12 committed:
Fix whitespace errors provided by stylint on Css file

ReyAntonio12 committed:
Fix line 198 whitespace error on style.css given by stylelint

Efiamotu-1 commented on index.html:
```diff
@@ -0,0 +1,53 @@
+  </header>
+  <!-- This section is the main part, containing heading and a brief description of my work-->
+  <main class="main__containerinfo">
+   <h1 id="titlePortfolio">Hey there. I’m Reynaldo. </h1>
```
- You are making use of the semantic `main` tag to represent the main section of the application which is very good practice but however on line 35 you created a `section` outside of this main tag, ideally the headline section which is what contains the main title and information about the developer shoulod be enclosed in a section tag along side the social media icons. for further explanations see the example below of the structure which should be adopted
```html
  <main class="main__containerinfo">
    <section>
      <h1 id="titlePortfolio">Hey there. I’m Reynaldo. </h1>
      <h2 id="developerPortfolio">I’m a software developer</h2>
      <p id="descriptionPortfolio"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
     don’t hesitate to contact me.</p>
      <ul class="container__icons">
        <li class="brand--icon">
          <a href="https://twitter.com/"><img src="images\brand-twitter-filled.png" alt="twitter logo" class="icon--image" /></a>
        </li>
      </ul>
    </section>
   </main>
```

Efiamotu-1 commented on index.html:
```diff
@@ -0,0 +1,53 @@
+    don’t hesitate to contact me.</p>
+  </main>
+
+  <section class="container__icons">
+    <div class="brand--icon">
+      <a href="https://twitter.com/"><img src="images\brand-twitter-filled.png" alt="twitter logo" class="icon--image" /></a>
+    </div>
+    <div class="brand--icon">
+      <a href="https://google.com"><img src="images\brand-gmail.png" alt="gmail logo" class="icon--image"></a>
+    </div>
+    <div class="brand--icon">
+      <a href="https://www.linkedin.com/in/reynaldo-casta%C3%B1o/?locale=en_US"><img src="images\brand-linkedin.png"alt="linkedin logo" class="icon--image"></a>
+    </div>
+    <div class="brand--icon">
+        <a href="https://github.com/ReyAntonio12"><img src="images\brand-github-filled.png" alt="github logo" class="icon--image"></a>
+    </div>
+    <div class="brand--icon">
+        <a><img src="images\angellist.png" alt="angellist logo" class="icon--image"></a>
+    </div>
+  </section>
```
- [ ] It is important to make use of semantic tags as much as possible in our application and as such i will suggest rather than using the non-semantic `div` tag as container for the social media icons, you make use of the semantic `ul` tag and each link would be wrapped in an `li` tag as a list item. see the example below for more reference :+1: 
```html
<ul class="socials">
  <li>
    <a href="https://github.com/"><img src="./assets/github.png" alt="github"/></a>
  </li>
  <li>
    <a href="https://linkedin.com/in/"><img src="./assets/linkedin.png" alt="linkedin"/></a>
  </li>
</ul>
```

Efiamotu-1 commented on index.html:
```diff
@@ -0,0 +1,53 @@
+  </main>
+
+  <section class="container__icons">
+    <div class="brand--icon">
```
- [ ] please ensure that the color for your social media icons are the same as the provided design on figma :+1: 
- figma design
![sm-i](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/e41fe20b-24ea-47ef-94ff-546cf2fd05b5)
- your design
![sm-ic](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/46fb03a0-d333-4922-a4ba-b9b34deb3fb8)

Efiamotu-1 commented on index.html:
- [ ] please as shown from your design the hamburger circled in red is overlapping the background image. can you kindly fix this design flaw so it match how it is supposed to look like from the figma design below. It is important that your design completely matches the figma template as provided in the design as this shows how much attention to details you pay as a skilled frontend developer :+1: :sparkles: 
- figma design
 ![fig](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/74338985-734c-4e91-8209-7b9aa36334f7)
- your design
![not](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/4d911b1d-7d6b-4510-8103-f8b5d5c3a69e)

ReyAntonio12 replied:
Hello @Efiamotu-1 

I have corrected most of the changes to the code but I'm still struggling with the hamburger icon. Is there a way to do it when using Flexbox I cannot get the desired result

This is my result at the end:
![image](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/110306452/8e06b15f-50e8-4bb1-b5df-0bd67362f2a0)

Thank you for your feedback.

Efiamotu-1 commented on index.html:
- [ ] It is also important to use good PR title and summary when making a pull request attached below is an image of how your pr title and summary for this project should look like please make sure to make the correct changes before resubmitting for a new pull request. :+1: 
![good-pr](https://user-images.githubusercontent.com/89055502/234389170-2ece21cc-1ad3-46e1-8e31-cb39f5f8304d.png)
- Don't mind that this is a pr for `hello world` project you should take note of how descriptive it is to tell you what was done on this pull request.

Efiamotu-1 reviewed (changes requested):
Hi @ReyAntonio12 , 

![](https://user-images.githubusercontent.com/89055502/221963929-dc172827-5c2c-411b-bf4d-923659f626e1.gif) 

Good job so far! 👏. Your Code is Awesome! 💪

There are some **few** issues 🐛 that you still need to work 🔧 on though to go to the next project ⏭️ but you are almost there! 🏃
## Don't Worry 🙏

It's ok to have bugs in your code 🐛. Please, see this code review as a **chance** for you to **learn** practical coding skills and learn **new tips** that can improve your code quality 💯 and help you **get a better ** 💼.

_I am going to guide you on how to fix them down below. 🤓_

## Highlights :dart: 
- ✔️ No linter Errors
- ✔️ Descriptive README
- The code is top-notch! 💯 and has a huge potential.
- please take note of these things for your next project milestone
- [ ] your branch name should be descriptive enough to describe the changes implemented for that branch alone :+1: 
- [ ] Yur PR title and description should as well be descriptive to explain what changes have been made for that milestone. please work on improving these skills as you progress and submit more milestone projects

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


## Important Notes 📝

- Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
- Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

> **Note:** Please, don't Merge This Pull Request yet. Only Merge when it is Approved ✅.


## Thank you for your time 🙏 and call me if you need help or clarification ☎️

- This is My **GitHub**: [![@Efiamotu-1](https://img.shields.io/github/followers/Efiamotu-1?label=Habeeb-Efiamotu-Musa&style=social)](https://github.com/Efiamotu-1/)
- This is My **Twitter**: [![@EFYAMOTU](https://img.shields.io/twitter/follow/omarramoun?label=Habeeb-Efiamotu-Musa&style=social)](https://twitter.com/EFYAMOTU)
- This is My **linkedIn**.[![Habeeb Efiamotu Musa](https://img.shields.io/badge/linkedin-@Habeeb-crimson)](https://www.linkedin.com/in/musa-habeeb/)
- This is My **Slack**: [![Habeeb Efiamotu Musa ](https://img.shields.io/badge/slack-@Habeeb-brightgreen)](https://slack.com)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Efiamotu-1 commented:
@ReyAntonio12  your deployed project for the live demo is also not working please it's either you fix the broken link as shown below or you remove that section since it is currently not a project requirement. :+1: 

![f-deploy](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/89055502/24572f86-2d2f-440e-934f-97c0669f9db8)

ReyAntonio12 commented:
- Linters Configuration
- Initial index.html added to the project
- Styles to main information and menu added
- Corrected bugs for semantic tags and positioning for menu button
- Changed name of pull request.

ReyAntonio12 committed:
Add new icons to social incons, add semantic tags to index and fixed positioning for mobile button for menu

Felix45 commented on index.html:
```diff
@@ -0,0 +1,54 @@
+</head>
+<body class="main__container">
+    <!-- Creating the navigation menu bar -->
+  <header class="container_menu">
+    <nav class="container--nav">
+        <ul class="navegation--list">
+            <li class="navigation--option"><a>Portfolio</a></li>
+            <li class="navigation--option"><a>About</a></li>
+            <li class="navigation--option"><a>Contact</a></li>
+        </ul>
+        <div class="menu--image">
+            <img src="images\menuUnion.png" alt="Menu button" id="menu--button"/>
+        </div>
+    </nav>
+  </header>
```
- [ Required ] I think it would be better if you made your navigation menu appear exactly as it is in the Figma design as requested by a previous reviewer, to achieve this I would like to suggest the following changes to your CSS
  
   - Remove the background image from the `body` tag and add it to the `main` container, then adjust the `CSS` 
   as shown below.

```CSS
/* remove the fixed positioning from the header */
.container_menu {
  padding: 24px 0; 
  width: 100%;
  display: flex;
  flex-direction: row-reverse;
}

/* Remove the fixed height and width from the nav */
.container--nav {
  width: 100%;
}

.main__containerinfo {
  padding-top: 100px;
  background-image: url('images/mobile_background.png');
  background-size: cover;
  background-repeat: no-repeat;
}
```

## Figma design 
![Screenshot from 2023-06-03 00-11-15](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/16392046/b6af0112-6969-4839-a14b-23933742e604)

## Your design 
![Screenshot from 2023-06-03 00-16-42](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/16392046/b85b525d-c8ac-4654-81ee-e4a17c58222c)

Felix45 reviewed (changes requested):
Hi @ReyAntonio12 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

<img src="https://user-images.githubusercontent.com/65860678/170491251-b1add8ba-b18f-4648-a893-20e510d957fd.gif" width="350px" />

 ## Highlights :heavy_check_mark: 
- [x] Good and correct use of semantic tags :white_check_mark: 
- [x]  Good UI design :white_check_mark: 
- [x] Linters are correctly setup :white_check_mark:  
- [x] Implemented some of the changes from the previous reviewer :white_check_mark:  

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

ReyAntonio12 committed:
Add changes to the responsive for cell phone, adjusted button icon

ReyAntonio12 commented:
Hello,

Please look at the following images:

![image1](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/110306452/ba4c2062-c75b-4c88-af27-19bfd6771690)
![image3](https://github.com/ReyAntonio12/Reynaldo--Portfolio/assets/110306452/bc0d1d22-936a-42ee-a667-8b970519c0f9)

I did not add the welcome word as it is not in the current template. 
I was able to fix the error you could point at me.

Thank you.

DuaneDave reviewed (approved):
Hi @ReyAntonio12 👋, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Very nice and clean UI
✔️ Clean and organized code

## Optional suggestions
N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you consider them as they can improve your code. Some of them were missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
